SWLOR.Game.Server/Feature/PerkDefinition/ForcePerkDefinition.cs

```diff
@@ -77,35 +77,35 @@ private void ThrowRock()
                 .Name("Throw Rock")
 
                 .AddPerkLevel()
-                .Description("Telekinetically launches a chunk of the environment at the enemy. Deals 10 physical DMG to a single target.")
+                .Description("Telekinetically launches a chunk of the environment at the enemy. Deals 10 + Willpower Mod * 2 physical DMG to a single target.")
                 .Price(1)
                 .RequirementSkill(SkillType.Force, 10)
                 .RequirementCharacterType(CharacterType.ForceSensitive)
                 .GrantsFeat(FeatType.ThrowRock1)
 
                 .AddPerkLevel()
-                .Description("Telekinetically launches a chunk of the environment at the enemy. Deals 15 physical DMG to a single target.")
+                .Description("Telekinetically launches a chunk of the environment at the enemy. Deals 20 + Willpower Mod * 3 physical DMG to a single target.")
```

Comment:
I'm really on the fence on giving exact math in the description. I feel it can cause confusion to players who don't care about it and just want to know what to raise to improve the scaling. It's a large part of the reason I chose to stick to a single "DMG" value - simplicity. 

The other thing is it's very error-prone. If you make a math change you also have to remember to come back here and update it.

I think I lean more toward simplifying this by saying something similar to "scales with WIL" or something like that. If we want exact numbers, those can go on the Wiki instead.

Thoughts?

Reply:
Yeah that works for me.